upgrade_test.py

```diff
@@ -664,10 +675,16 @@ def test_rolling_upgrade(self):  # pylint: disable=too-many-locals,too-many-stat
         self.metric_has_data(
             metric_query='sct_cassandra_stress_write_gauge{type="ops", keyspace="keyspace1"}', n=5)
 
-        # start gemini write workload
-        if self.version_cdc_support():
+        gemini_cmd = self.params.get("gemini_cmd")
+        gemini_thread = None
+
+        if gemini_cmd:
             InfoEvent(message="Start gemini during upgrade").publish()
-            gemini_thread = self.run_gemini(self.params.get("gemini_cmd"))
+            # TODO: workaround for issue #16317
+            if self.version_cdc_support() and not self.tablets_enabled:
+                gemini_cmd += " --table-options \"cdc={'enabled': true}\""
```

Comment:
I don't understand what's the point of running gemini without cdc ?

if the whole purpose it was introduce was for testing cdc...

Reply:
It's not the whole point of running Gemini.
Let's not remove it.

Reply:
a bit change the code but save the main idea, to run gemini during rolling upgrade with and without cdc support